### PR TITLE
Adds documentation about name templates in grid editors

### PR DESCRIPTION
**Pull request by skttl**
If https://github.com/umbraco/Umbraco-CMS/pull/6634 gets merged, this PR adds documentation for the feature.

### CODE REVIEW

**Comment by skttl** [comment]
We probably should, wasn't sure how it is organized :)

**Comment by sofietoft** [comment]
No worries 😄 
It's not the most intuitive versioning system we've set up, but it does the trick 💪

**Comment by sofietoft** [comment]
We're closing down Hacktoberfest, so just wanted to let you know, that I've added the label to your PR 😁 

Of we can't merge until Umbraco 8.3 is released, so won't do more with it today.
Oh, and would you be willing to make the versioning changes we talked about? I'll be happy to do it as well of course - would just be nice to have it all in one PR if possible.
Do let me know 😁

**Comment by jmayntzhusen** [comment]
This change has been moved to 8.4, FYI 😉

**Comment by sofietoft** [comment]
@skttl - I'm having a hard time figuring out if this one was added to the Umbraco 8.4 release, that went live yesterday.
Would you be able to help me out? 😅

**Comment by jmayntzhusen** [comment]
https://github.com/umbraco/Umbraco-CMS/issues/6633 This is the change, it is in 8.4.
I know because @skttl showed it at last nights meetup 😄

**Comment by sofietoft** [comment]
Ah, great, thanks @jmayntzhusen ! 😄 

@skttl - should I take care of fixing the versioning, or would you be up for it? 😁

**Comment by skttl** [comment]
Changed, and I can double verify that the change is in 8.4 - just tried it out for my new version of DTGE :)

**Comment by sofietoft** [comment]
Ah, I actually meant making separate articles, so we keep the one for Umbraco 7!
Sorry, that's my bad for not being clear about it 🙈 

Would you be up for renaming the old article to index-v7.md with `versionFrom: 7.0.0`, and then creating a new index.md file with `versionFrom: 8.0.0` plus the new lines you're adding here?

File: Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Grid-Layout/Grid-Editors.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-versionFrom: 7.0.0
+versionFrom: 8.4.0
```
Comment (sofietoft) [inline Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Grid-Layout/Grid-Editors.md]:
```suggestion
versionFrom: 7.0.0
```

File: Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Grid-Layout/Grid-Editors.md [hunk 2]
```diff
@@ -62,6 +62,13 @@ For a grid editor, the required values are:
 - `icon`: Icon shown to the editor, uses same icon classes as the rest of
 - `view` the view defines the editor used to enter a value. By default Umbraco will look in `/umbraco/views/propertyeditors/grid/editors` for a html view to use - but you can pass in your own path
 
+You can also add a name template for generating grid item labels using the syntax `{{ value.propertyAlias }}`.
```
Comment (sofietoft) [inline Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Grid-Layout/Grid-Editors.md]: I'm just adding this as a "tip" instead:

```suggestion
:::tip
In **Umbraco 8.4** you can also add a name template for generating grid item labels using the syntax `{{ value.propertyAlias }}`.
```

File: Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Grid-Layout/Grid-Editors.md [hunk 3]
```diff
@@ -62,6 +62,13 @@ For a grid editor, the required values are:
...
+
+- If you would like to include the index position in the label, you can use `{{$index}}`.
+- If your editor links to a content, media or member node, you can use the Angular filter `{{ value.udi | ncNodeName }}` to show the node name rather than the node ID.
+- If your editor is a rich text editor, you can use the Angular filter `{{ value | ncRichText }}` to show the unformatted text.
```
Comment (sofietoft) [inline Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Grid-Layout/Grid-Editors.md]:
```suggestion
- If your editor is a rich text editor, you can use the Angular filter `{{ value | ncRichText }}` to show the unformatted text.
:::
```

**Comment by skttl** [comment]
Much better :)

I think the version should be 7.2.0 as that is the version that introduced grid.

**Comment by sofietoft** [comment]
Ah, good point @skttl 😁 
It's been updated.

Would you mind committing the suggestions?
I don't have permission to do it 😅

**Comment by sofietoft** [comment]
@skttl , the PR for Core has been merged and released, right?

Do you have time to commit the suggestions I've made to your PR here?
It's OK if you don't, then I'll make the changes myself 😄 
Just let me know. Thanks!

**Comment by skttl** [comment]
Sorry :)

**Comment by sofietoft** [comment]
No worries at all @skttl ! 😄 
Thanks. I'll get this merged 💪

**Pull request merged**